scripts/govuk_app_owners.coffee

```diff
@@ -11,3 +11,5 @@ module.exports = (robot) ->
         if response.statusCode == 200
           data = JSON.parse(body)
           res.reply "#{data.app_name} is owned by #{data.team}"
+        else
```

Comment:
I didn't do this initially, because of the risk of false positives: https://github.com/alphagov/gds-hubot/pull/21#discussion_r127155482